### PR TITLE
Make the TAC repo into a site

### PR DESCRIPTION
**Pull request by jmertic**
Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

### CODE REVIEW

**Comment by jfpanisset** [comment]
Could you provide some background on what this does / how it's supposed to work going forward? I'm guessing the goal is to provide a friendlier web interface than just the GitHub "Code" view, but for instance it seems that (every?) Markdown document now has a HTML version, which brings up a couple of questions:

* is this necessary or could it be done on the fly (especially since this isn't going to be high traffic)
* will HTML versions of Markdown documents be auto generated at PR merge time, or will additions to the tac repo require generating these as part of the PR?

**Comment by jmertic** [comment]
Hey @jfpanisset and @lgritz !

Good questions. You are right that the intent here is to make the display a bit more friendly, and there are some backend automation bits that I'm looking to employ to keep things like the TAC voting member list up to date.

To your specific questions:

- This is using Jekyll, so yes all the translation from Markdown to HTML is done automatically on every commit merged.
- For PRs there will be a preview build done so you can see what it looks like before merging, but again there is no additional work needed for you.

The one thing I'm trying to figure out a clever solution for is building the agenda doc with the current list of TAC voting members, but let me chew on that and I'll get something going.

Let me know if this helps!

**Review by jfpanisset — APPROVED** [review]
LGTM, might be good to have a top-level README_JEKYLL.md or something like that explaining briefly how this works, for instance:
- the checks and actions that get done when a PR is submitted / merged
- the fact that _site is mostly a pre-computed cache, and that files in there should not be manipulated directly
- metadata requirements at the top of individual Markdown files (the parent: and name: properties)

This can be addressed in a followup PR.

**Comment by jmertic** [comment]
Good points @jfpanisset! I removed the `_site` directory since that's autogenerated. I'll also add a few notes for where files would be added to ensure folks add the right metadata as well.

Perhaps let's show this at the TAC meeting Wednesday and then merge in thereafter?

File: index.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+# Technical Projects
+
+Technical Projects are approved by TAC Members per the [ASWF Project Lifecycle guidelines](https://github.com/AcademySoftwareFoundation/tac/blob/master/process/lifecycle.md). You can submit your project for inclusion as an ASWF Technical Project by completing the [Project Contribution Proposal Template](https://github.com/AcademySoftwareFoundation/tac/blob/master/process/proposal_template.md) and emailing it to tac@lists.awsf.io.
+
+Each of the ASWF projects are open to participation by anyone subject to the governance each project has adopted. If you are looking for a way to contribute to a project, many ASWF projects maintain a list of "good first issues" ( linked to below ).
+
+Current Technical Projects are listed below.
+
+<!-- Embed list of all Open Mainframe Project members -->
```
Comment (jfpanisset) [inline index.md:52]: This comment looks like cut-n-paste from Open Mainframe Project

File: index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+There are many more VFX related open source projects than what is hosted at ASWF; check out the list and add any we are missing at the [ASWF Landscape](https://landscape.aswf.io)
+
+[![ASWF Landscape](https://landscape.aswf.io/images/landscape.png)](https://landscape.aswf.io)
+
+# Current Working Groups
+
+Working groups may be formed by the TAC to focus on specific areas of technical interest or need within the ASWF.  They report to the TAC which remains responsible for any formal decisions or voting matters.
+
+## Active
+
```
Comment (jfpanisset) [inline index.md:66]: There are some missing WGs:

- Review and Approval
- Diversity and Inclusion
- Assets

File: _data/tacmembers.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
+Full Name,Account Name: Account Name,Appointed By,Voting Status,Special Role,Title,Email,Bio
```
Comment (jfpanisset) [inline _data/tacmembers.csv:1]: Although the CSV source document is public and contains this information, I'm not sure it's a good idea to commit personal / work emails for all TAC members to the repo? At least not before asking people whether they are OK with that or not.

File: index.md (hunk 1 above)
Comment (jmertic) [reply index.md]: Fixed!

File: index.md (hunk 2 above)
Comment (jmertic) [reply index.md:66]: I'll add D&I, but the other two haven't been approved yet so technically aren't a WG yet.

File: _data/tacmembers.csv (hunk 3 above)
Comment (jmertic) [reply _data/tacmembers.csv:1]: Seems fair, we can control this.

**Review by jfpanisset — COMMENTED** [review]
I'm iffy on whether the "source of truth" for TAC membership should be an external Google Docs spreadsheet, that seems like a somewhat brittle mechanism, although I guess the worse that happens if that link breaks is that the local cache tacmembers.csv doesn't get updated. I'm more concerned about having personal / work emails committed to the repo.

**Comment by jmertic** [comment]
> I'm iffy on whether the "source of truth" for TAC membership should be an external Google Docs spreadsheet, that seems like a somewhat brittle mechanism, although I guess the worse that happens if that link breaks is that the local cache tacmembers.csv doesn't get updated. I'm more concerned about having personal / work emails committed to the repo.

We can remove the emails in the csv, as they aren't used anyways.

For the Google Doc, it's an in-between of the internal systems that track this and this site. That sheet is accurate from our internal records.

**Pull request merged**